webapp/backend/pkg/thresholds/ata_attribute_metadata.go

```diff
@@ -662,13 +667,33 @@ var AtaMetadata = map[int]AtaAttributeMetadata{
 	188: {
 		ID:          188,
 		DisplayName: "Command Timeout",
-		DisplayType: AtaSmartAttributeDisplayTypeRaw,
+		DisplayType: AtaSmartAttributeDisplayTypeTransformed,
 		Ideal:       ObservedThresholdIdealLow,
 		Critical:    true,
 		Description: "The count of aborted operations due to HDD timeout. Normally this attribute value should be equal to zero.",
+		Transform: func(normValue int64, rawValue int64, rawString string) int64 {
+			// Parse Seagate command timeout values if the string contains 3 pieces
+			// and each piece is less than or equal to the next (as a sanity check)
+			// See https://github.com/AnalogJ/scrutiny/issues/522
+			pieces := strings.Split(rawString, " ")
+			if len(pieces) == 3 {
+				int_pieces := make([]int, len(pieces))
+				var err error
+				for i, s := range pieces {
+					int_pieces[i], err = strconv.Atoi(s)
+					if err != nil {
+						return rawValue
+					}
+				}
+				if int_pieces[2] >= int_pieces[1] && int_pieces[1] >= int_pieces[0] {
```

Comment:
This logic is slightly flawed. The first number indicates total command timeouts, the second indicates all commands that took over 5 seconds to complete, and the third all commands that took over 7.5 seconds to complete.

See:
```
Raw Usage

Raw [1 - 0] = Total # of command timeouts, with Max hold of FFFFh

Raw [3 - 2] = Total # of commands with > 5 second completion, including those > 7.5 seconds

Raw [5 - 4] = Total # of commands with > 7.5 second completion
```

Reply:
It's actually correct. When the string gets split, the indices are backwards from the byte order, and the int pieces variable uses that same backwards ordering

Reply:
@kaysond Thank you for the clarification!